documentation/src/main/doc/modules/ROOT/pages/getting-started.adoc

```diff
@@ -79,15 +79,18 @@ Running the previous example should give the following output:
 
 [NOTE]
 ====
-SmallRye Reactive Messaging uses SLF4J as logger.
-Check the https://www.slf4j.org/manual.html[SLF4J documentation] to configure the logs.
+SmallRye Reactive Messaging uses JBoss-logger as logger.
```

Comment:
JBoss logging no? 

Can you link to the project documentation page as it's not as known as SLF4J.

Reply:
I am not sure what is the correct project URL....
This (https://github.com/jboss-logging/jboss-logging) does not have any documentation.

Reply:
@dmlloyd Do you know where we can find the jboss-logging documentation?

Reply:
@phillip-kruger can you change to Jboss Logging at least?

Reply:
still JBoss-logger?

Reply:
No I don't.  There is some old Javadoc published at jboss.org.  Maybe @jamezp knows more?

Reply:
There is currently no documentation for the logging facade itself. There is some documentation for the logging tooling https://jboss-logging.github.io/jboss-logging-tools/. We could publish the JavaDoc's though.

Reply:
Thanks @jamezp! That's what we were looking for. @phillip-kruger can you add a link to https://jboss-logging.github.io/jboss-logging-tools/?